packages/tldraw/src/lib/shapes/arrow/ArrowShapeUtil.tsx

```diff
@@ -323,7 +334,7 @@ export class ArrowShapeUtil extends ShapeUtil<TLArrowShape> {
 		return update
 	}
 
-	override onTranslateStart: TLOnTranslateStartHandler<TLArrowShape> = (shape) => {
+	override onTranslateStart(shape: TLArrowShape) {
```

Comment:
Losing the types here for the ShapeUtil methods is unfortunate, because it means all of the methods need to manually type the parameters. TypeScript doesn't make this very easy.

Reply:
Autocomplete is quite good in these cases, it can infer the types in the suggestions:

https://github.com/user-attachments/assets/57caf2c8-80b8-4fcc-8261-db4f46c2bdac